[2021-03-10 06:59:53,365] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: data_processing.processing_user 2021-01-06T00:00:00+00:00 [queued]>
[2021-03-10 06:59:53,372] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: data_processing.processing_user 2021-01-06T00:00:00+00:00 [queued]>
[2021-03-10 06:59:53,372] {taskinstance.py:1017} INFO - 
--------------------------------------------------------------------------------
[2021-03-10 06:59:53,372] {taskinstance.py:1018} INFO - Starting attempt 1 of 1
[2021-03-10 06:59:53,372] {taskinstance.py:1019} INFO - 
--------------------------------------------------------------------------------
[2021-03-10 06:59:53,380] {taskinstance.py:1038} INFO - Executing <Task(PythonOperator): processing_user> on 2021-01-06T00:00:00+00:00
[2021-03-10 06:59:53,382] {standard_task_runner.py:51} INFO - Started process 42080 to run task
[2021-03-10 06:59:53,387] {standard_task_runner.py:75} INFO - Running: ['airflow', 'tasks', 'run', 'data_processing', 'processing_user', '2021-01-06T00:00:00+00:00', '--job-id', '48', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/data_processing.py', '--cfg-path', '/tmp/tmp59_iq9sn']
[2021-03-10 06:59:53,388] {standard_task_runner.py:76} INFO - Job 48: Subtask processing_user
[2021-03-10 06:59:53,529] {logging_mixin.py:103} INFO - Running <TaskInstance: data_processing.processing_user 2021-01-06T00:00:00+00:00 [running]> on host airflowvm
[2021-03-10 06:59:53,664] {taskinstance.py:1230} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=data_processing
AIRFLOW_CTX_TASK_ID=processing_user
AIRFLOW_CTX_EXECUTION_DATE=2021-01-06T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-01-06T00:00:00+00:00
[2021-03-10 06:59:53,673] {python.py:118} INFO - Done. Returned value was: None
[2021-03-10 06:59:53,676] {taskinstance.py:1135} INFO - Marking task as SUCCESS. dag_id=data_processing, task_id=processing_user, execution_date=20210106T000000, start_date=20210310T065953, end_date=20210310T065953
[2021-03-10 06:59:53,693] {taskinstance.py:1195} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-03-10 06:59:53,722] {local_task_job.py:118} INFO - Task exited with return code 0
